packages/material-ui/src/locale/index.js

```diff
@@ -0,0 +1,170 @@
+export const deDE = {
+  props: {
+    MuiTablePagination: {
+      backIconButtonText: 'Nächste Seite',
+      labelRowsPerPage: 'Filas por página:',
+      labelDisplayedRows: ({ from, to, count }) => `${from}-${to === -1 ? count : to} of ${count}`,
+      nextIconButtonText: 'Siguiente página',
+    },
+    MuiRating: {
+      getLabelText: value => `${value} ${value !== 1 ? 'Sterne' : 'Star'}`,
+    },
+    MuiAutocomplete: {
+      clearText: 'Klar',
```

Comment:
```suggestion
      clearText: 'Leeren',
```

that was basically translated as "Ok!" or "Roger!" :smile: